docs/en/Core-Angular-Sign-In-Without-Specifying-Tenant.md

```diff
@@ -0,0 +1,313 @@
+## Sign In Without Specifying Tenant
+
+Normally, **ASP.NET Zero** uses tenant information in login transactions. This document shows you how to implement tenant information independent login. 
+
+***Important Note:*** *To implement that, your user's email addresses have to be unique. Otherwise, that solution may not works properly.*
```

Comment:
```suggestion
***Important Note:*** *Your user's email addresses have to be unique to implement this solution. Otherwise, this solution may not work correctly.*
```